detection-rules/link_content_credential_phishing.yml

```diff
@@ -39,7 +39,20 @@ source: |
       and not profile.by_sender().any_false_positives
     )
   )
-
+  
+  // negate docusign 'via' messages
+  and not (
+      any(headers.hops,
+          any(.fields, .name == "X-Api-Host" and strings.ends_with(.value, "docusign.net"))
+      )
+      and strings.contains(sender.display_name, "via")
+    )
```

Comment:
i think indentation is wrong here

Reply:
Does it look better now? This is how the formatter formatted it :)